cmd/create/machinepool/helper.go

```diff
@@ -0,0 +1,81 @@
+package machinepool
+
+import (
+	"os"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/openshift/rosa/pkg/aws"
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/rosa"
+	"github.com/spf13/cobra"
+)
+
+func getSubnetFromUser(cmd *cobra.Command, r *rosa.Runtime, isSubnetSet bool,
+	cluster *cmv1.Cluster) string {
+	var selectSubnet bool
+	var subnet string
+	var err error
+
+	question := "Select subnet for a single AZ machine pool"
+	questionError := "Expected a valid value for select subnet for a single AZ machine pool"
+
+	if cluster.Hypershift().Enabled() {
+		question = "Select subnet for a hosted machine pool"
+		questionError = "Expected a valid value for select subnet for a hosted machine pool"
```

Comment:
"Expected a valid value for select subnet for a hosted machine pool" 

shouldnt this be 

""Expected a valid value for subnet for a hosted machine pool